cdap-proto/src/test/java/io/cdap/cdap/proto/app/AppUpdateGitMetaRequestTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.proto.app;
+
+import com.google.gson.Gson;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AppUpdateGitMetaRequestTest {
+  private static final Gson GSON = new Gson();
+
+  /**
+   * Test {@link AppUpdateGitMetaRequest} deserialization
+   */
+  @Test
+  public void testAppMarkLatestRequestDeserialize() throws Exception {
+    String appUpdateGitMetadataRequest = "{\n"
+        + "  \"appId\": \"test-application-id\","
```

Comment:
as mentioned in the other PR, this is not a useful test that we need to perform, as it is basically just testing that Gson does the right thing.

Instead, there should be unit tests covering the new logic that was added.

Reply:
removed this test.